### DISCUSSION TITLE
Executing sudo command on Linux

### DISCUSSION THREAD

**Discussion by qaniqani** [Tyrrrz/CliWrap#156] [Q&A]
When I want to execute command on sudo in Linux it is waiting for password.
Can we send the password as a parameter?

**Comment by Tyrrrz** [comment]
If it's waiting for it in stdin, then you can push the password through stdin.

**Reply by qaniqani** [reply]
I am using as below, but no result.

var cmd = Cli.Wrap("sudo")
             .WithArguments(input.Command)
             .WithValidation(CommandResultValidation.None)
             .WithStandardInputPipe(PipeSource.FromString("XXXX"));

Can you see any errors here?

Thnx

**Reply by Tyrrrz** [reply]
Looks right to me. Unfortunately, I don't have enough experience with Linux/sudo to advise exactly how to achieve what you want. Another option may be to run the main process via sudo, then perhaps all child processes will spawn with appropriate privileges as well.

**Reply by qaniqani** [reply]
Thanks for your help. I will try your suggestion.
Note: As the people of Turkey, we stand by Ukraine. <3 Bayraktar
